### ISSUE TITLE
IEV data anomaly (20201217): 103-02-04 REPLACES is malformed slightly

### ISSUE THREAD

**Issue by skalee** [glossarist/iev-data#150]
Concept `103-02-04` supersedes `103-02-04:2009:12`. There are no other entries like this one. It looks like a typo, I suppose it should be `103-02-04:2009-12` instead. But maybe we should ask for clarification first, what do you think @ronaldtse? Obviously it won't be a big deal to fix it on our end if this is preferred.

**Comment by ronaldtse** [comment]
Yes I agree that `103-02-04:2009:12` should have been `103-02-04:2009-12`. This is a source data issue.
